### PR TITLE
Conn 1074

### PR DESCRIPTION
**Pull request by alameluchidambaram**

### CODE REVIEW

File: Product/Production/Adapters/General/gateway.properties [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Duration in seconds between file reload
```
Comment (msweaver) [inline Product/Production/Adapters/General/gateway.properties]: why is this here?

**Comment by msweaver** [comment]
There are a number of formatting issues with the poms specifically.

File: Product/Production/Adapters/General/gateway.properties (hunk 1 above)
Comment (alameluchidambaram) [inline Product/Production/Adapters/General/gateway.properties]: Gateway.properties is added accidently here while dragging in windows explorer.will remove it from codebase.

**Comment by alameluchidambaram** [comment]
Fixed all the formatting pom.xml issues using Eclipse.
